pii_check/pii_check_hook.py

```diff
@@ -22,8 +22,12 @@ def get_payload(content, enabled_entity_list, blocked_list):
 
     if blocked_list:
         blocked_list_payload = []
-        for block_pattern in blocked_list:
-            curr_block_item = {"type": "BLOCK", "value": block_pattern}
+        if (blocked_list % 2) != 0:
+            sys.exit("Uneven number of blocked list parameters. Please provide parameters as 'ENTITY TYPE' - 'PATTERN' pairs in config")
```

Comment:
```suggestion
            sys.exit("Uneven number of blocked list parameters. Please provide parameters as space-separated 'ENTITY TYPE' 'PATTERN' pairs in config")
```
I am afraid that users think that we need a dash between the two entries.